### PR TITLE
[WIP] feat(debug): add debug flags to profile mgmt pods

### PR DESCRIPTION
**Pull request by kmova**
Support ENV variables to enable CPU profiling.

### CODE REVIEW

File: pkg/env/v1alpha1/env_test.go [hunk 1]
```diff
@@ -131,3 +131,48 @@ func TestgetEnv(t *testing.T) {
...
+			expectValue: false,
+		},
+	}
+	for tname, tc := range testCases {
+		t.Run(tname, func(t *testing.T) {
+			os.Setenv(string(tc.key), tc.value)
+			defer os.Unsetenv(string(tc.key))
+
+			feature := Matches(tc.key, tc.pattern)
+			if !reflect.DeepEqual(feature, tc.expectValue) {
```
Comment (AmitKumarDas) [inline pkg/env/v1alpha1/env_test.go]: Let us avoid reflect wherever possible.

File: pkg/install/v1alpha1/cstor_pool.go [hunk 2]
```diff
@@ -308,6 +315,10 @@ spec:
               # OPENEBS_IO_CSTOR_ID env has UID of cStorPool CR.
             - name: OPENEBS_IO_CSTOR_ID
               value: {{ pluck .ListItems.currentRepeatResource .ListItems.nodeUidMap.nodeUid |first | splitList " " | first}}
+            {{- if ne $debugProfileVal "false" }}
+            - name: "OPENEBS_IO_DEBUG_PROFILE"
+              value: "{{- $debugProfileVal }}"
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_pool.go]: I guess this is syntactically wrong. Valid values can be following:
```yaml
value: {{ $debugProfileVal | quote }}
```
I am not sure if quote is required!!

File: pkg/install/v1alpha1/cstor_volume.go [hunk 3]
```diff
@@ -414,6 +421,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            {{- if ne $debugProfileVal "false" }}
+            - name: "OPENEBS_IO_DEBUG_PROFILE"
+              value: "{{- $debugProfileVal }}"
```
Comment (AmitKumarDas) [inline pkg/install/v1alpha1/cstor_volume.go]: Same comments as above... Please check if the syntax is correct.

File: ci/install_openebs.sh [hunk 4]
```diff
@@ -25,7 +25,9 @@ sudo docker tag openebs/cstor-pool-mgmt:ci quay.io/openebs/cstor-pool-mgmt:${CI_
 sudo docker tag openebs/cstor-volume-mgmt:ci quay.io/openebs/cstor-volume-mgmt:${CI_TAG}
 
 
-kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/${CI_BRANCH}/k8s/openebs-operator.yaml
+#kubectl apply -f https://raw.githubusercontent.com/openebs/openebs/${CI_BRANCH}/k8s/openebs-operator.yaml
+#TODO Use the following to overwrite to stable images
+kubectl apply -f $GOPATH/src/github.com/openebs/maya/ci/yamls/openebs-operator.yaml
```
Comment (ashishranjan738) [inline ci/install_openebs.sh]: Are we going to maintain two operators with this ?

**Comment by AmitKumarDas** [comment]
Closing this since this has become stale

**Pull request closed without merging**